src/driver.rs

```diff
@@ -98,6 +108,42 @@ impl Driver {
         Ok(unsafe { Dataset::from_c_dataset(c_dataset) })
     }
 
+    pub fn create_with_band_type_with_options<T: GdalType>(
+        &self,
+        filename: &str,
+        size_x: isize,
+        size_y: isize,
+        bands: isize,
+        options: Vec<RasterCreationOption>,
```

Comment:
I think this can be `options: &[RasterCreationOption]`.

Reply:
Yes, I agree. Looks much better!